lib/elixir/lib/option_parser.ex

```diff
@@ -499,4 +562,28 @@ defmodule OptionParser do
   defp negative_number?(arg) do
     match?({_, ""}, Float.parse(arg))
   end
+
+  defp format_errors(errors, opts) do
+    details = Enum.map_join(errors, ". ", &format_error(&1, opts))
+    total = Enum.count(errors)
+    error = if total == 1, do: "error", else: "errors"
+    "#{total} #{error} found! #{details}"
```

Comment:
No worries about those. My plan is to merge the OptionParser related PRs this week and I will fix any inconsistency (specially because those PRs have been blocking on me for a while). :)